src/Compilers/CSharp/Portable/Symbols/MemberSignatureComparer.cs

```diff
@@ -313,7 +314,7 @@ internal sealed class MemberSignatureComparer : IEqualityComparer<Symbol>
             considerReturnType: true,
             considerTypeConstraints: true,
             considerCallingConvention: true,
-            considerRefKindDifferences: true,
+            refKindCompareMode: RefKindCompareMode.ConsiderDifferences,
```

Comment:
>refKindCompareMode: RefKindCompareMode.ConsiderDifferences, [](http://example.com/codeflow?start=12&length=59)

Are we testing CSharpSignatureAndConstraintsAndReturnTypeComparer?

Reply:
No, as the note above it says, it's not used anywhere.

Reply:
> No, as the note above it says, it's not used anywhere.

Let's delete it then.